governance/mission-values.md

```diff
@@ -1,7 +1,6 @@
 # Mission and values
 
-The mission of pyOpenSci is to support diverse community around the free, open
-python tools that drive open, reproducible scientific workflows.
+The mission of pyOpenSci is to support building a diverse community of practice around the free, open Python tools that drive open, reproducible scientific workflows.
```

Comment:
"support building" sounds awkward to me. I like "support" on its own, or if you wanted something more active, you could say "build and support". Or "expand and support a diverse" to emphasize the theme of building on existing work?

Reply:
Also, on https://www.pyopensci.org/handbook/governance/index.html we state the mission as "pyOpenSci is an non-profit organization whose mission is to build diverse, supportive community around the free and open Python tools that drive open science." The mission statement should be the same in all places.